File: problem1.js
var multiples = []
var sum = 0;


// find multiples of 3 or 5
for (var i = 0; i < 1000; i++) {
  if (i % 3 === 0 || i % 5 === 0) {
    multiples.push(i);
  };
}

// sums up numbers 
for (i=0; i<multiples.length; i++)  {
  sum = sum + multiples[i];
}

File: problem1.rb
# multiples = (0...1000).select do |i|
#   i % 3 === 0 || i % 5 === 0
# end

# multiples.inject{|sum,x| sum + x }

def calculate(min, max, num1, num2)
  multiples = (min...max).select do |i|
    i % num1 === 0 || i % num2 === 0
  end

  multiples.inject{|sum,x| sum + x }
end